client/theme.js

```diff
@@ -248,6 +254,9 @@ export default {
   ...baseThemes,
   [Theme.contrast]: extend(baseThemes[Theme.dark], {
     inactiveTextColor: grays.light,
+    modalBorderColor: grays.middleDark,
+    searchBackgroundColor: grays.darker,
+    tableRowStripeColor: grays.dark,
```

Comment:
This isn't necessary due to the `extend(baseThemes[Theme.dark]`.  We put that in as a helped to avoid having to repeat values which are the same as the dark theme.

Reply:
Deleted redundant section.Thanks for pointing it out!